src/types.ts

```diff
@@ -33,11 +25,18 @@ export type OpenRouterSharedSettings = {
       }
   );
 
-  extraBody?: Record<string, any>;
-
   /**
-A unique identifier representing your end-user, which can help OpenRouter to
-monitor and detect abuse.
+   * A unique identifier representing your end-user, which can
+   * help OpenRouter to monitor and detect abuse.
    */
   user?: string;
 };
+
+export type OpenRouterSharedSettings = OpenRouterProviderOptions & {
```

Comment:
do you know the history behind this split of OpenRouterProviderOptions & OpenRouterSharedSettings? like whats the motivation for splitting these fields out into another sub type, are these just unofficial/nonstandard fields?

Reply:
All unofficial fields